Binary_search/Simple_search.py

```diff
@@ -0,0 +1,29 @@
+
+'''
+n, k = [int(i) for i in input().split()]
+
+N = [int(i) for i in input().split()]
+K = [int(i) for i in input().split()]
+'''
+
+
+def Simple_search(Arr, search):

```

Comment:
Simple is subjective :) Let's call it linear
```suggestion
def linear_search(array, number):
```